connector/connect/server/src/main/scala/org/apache/spark/sql/connect/planner/SparkConnectPlanner.scala

```diff
@@ -352,6 +353,16 @@ class SparkConnectPlanner(session: SparkSession) {
       data = Tuple1.apply(cov) :: Nil)
   }
 
+  private def transformStatCorr(rel: proto.StatCorr): LogicalPlan = {
+    val corr = Dataset
+      .ofRows(session, transformRelation(rel.getInput))
+      .stat
+      .corr(rel.getCol1, rel.getCol2, rel.getMethod)
```

Comment:
what if the optional `method` is not set ?

Reply:
I think we need to check `rel.hasMethod`

Reply:
The default value is `pearson`

Reply:
in this server side we need to check `rel.hasMethod` since it is optional.

other clients can not set `pearson` according to the definition of this proto message

Reply:
I got it.